### DISCUSSION TITLE
Pairing device :Aqara N100 Smart Door Lock

### DISCUSSION THREAD

**Discussion by jbenabdallah** [Koenkk/zigbee2mqtt#6143] [General]
Hi,
I would like to pair my Aqara N100 Smart Door Lock with zigbee2mqtt
I can't find the instruction how to pair it. 
Any one knows how to pair this device ?

Thanks !

**Comment by johnjoemorgan** [comment]
Mine just arrived.  My plan was to integrate it with Z2M as well.  Looking at this I think it's only compatible with the Aqara Zigbee Coordinators.  I have an E1 on order now.

Using Homekit I can get the lock on to an Apple device but I can't get that accessory into HA.  So at the moment I have 2 expensive paper wieghts 

I'll comment back when I get the Aqara E1

**Reply by Rob-Powell** [reply]
any update John?

**Reply by nevir** [reply]
This thread may be helpful: https://community.home-assistant.io/t/how-to-aqara-zigbee-devices/282845

**Reply by almirus** [reply]
I have E1 and n100, it isn't work for zigbee2mqtt
also doesn't work with HomeKit via HA
